example/custom-plugin/README.md

```diff
@@ -0,0 +1,22 @@
+# Build plugin
+
+## In docker
+Plugin must use `musl-gcc` to build. The best choose is build it in the docker. 
```

Comment:
hm... IMO, the compiler of plugin and scheduler should be the same; as some users may build scheduler themself.

Reply:
because we use alpine as base image，which only has musl-libc

Reply:
yes, that's ok in our case. But if someone want to build scheduler themself, it's better to highlight that; or add add a reference about the limitation of `Plugin`.